libs/api/domains/license-service/src/lib/licenseService.service.ts

```diff
@@ -198,6 +210,9 @@ export class LicenseServiceService {
         locale,
       )
 
+      const metricsTags = this.getMetricsTags(license.type)
+      this.metrics.increment('all_licenes.license', metricsTags)
```

Comment:
Some of these metrics feel a bit redundant with the enhanced fetch metrics (timings and success/failure), but I guess they make it easier to assemble a focused "license dashboard".

Reply:
This makes it more explicit when we create&verify license and succeed/fail doing so. It will help in creating an explicit dashboard for the licenses which we've wanted for some time.

But I'm not experienced enough with this in production to know if this is the best way to do it, any suggestions welcome!